go/chat/teams.go

```diff
@@ -360,15 +360,27 @@ func LoadTeam(ctx context.Context, g *libkb.GlobalContext, tlfID chat1.TLFID, tl
 		if err != nil {
 			return team, err
 		}
-		team, err = teams.Load(ctx, g, ltarg(teamID))
-		if err != nil {
-			return team, err
+		loadAttempt := func(repoll bool) error {
+			arg := ltarg(teamID)
+			arg.ForceRepoll = arg.ForceRepoll || repoll
+			team, err = teams.Load(ctx, g, arg)
+			if err != nil {
+				return err
+			}
+			if !tlfID.EqString(team.KBFSTLFID()) {
+				return ImpteamUpgradeBadteamError{
+					Msg: fmt.Sprintf("mismatch TLFID to team: %s != %s", team.KBFSTLFID(), tlfID),
+				}
+			}
+			return nil
 		}
-		if !tlfID.EqString(team.KBFSTLFID()) {
-			return team, ImpteamUpgradeBadteamError{
-				Msg: fmt.Sprintf("mismatch TLFID to team: %s != %s", team.KBFSTLFID(), tlfID),
+		if err = loadAttempt(false); err != nil {
+			g.Log.CDebugf(ctx, "++Chat: LoadTeam: failed to load without repoll, trying again: %s", err)
+			if err = loadAttempt(true); err != nil {
+				return team, err
 			}
 		}
+
```

Comment:
This is ok but will double any other failure (like network conditions, or something actually wrong). Could do this to only loop on the condition you care about.
```
for repoll := range []bool{false, true} {
  ...
  arg.ForceRepoll = arg.ForceRepoll || repoll
  ...Load...
  if err != nil {
    return team, err
  }
  if !tlfID.EqString(team.KBFSTLFID()) {
    log("trying again")
    continue
  }
}
```